### PR TITLE
[GOV-570] New Batch and Voucher creation views

### PR DESCRIPTION
**Pull request by alberto-art3ch**
## Description

[GOV-570] Create Voucher Screen, New Batch and Voucher creation views

Screenshot 2024-03-06 at 9 52 27 p m

Screenshot 2024-03-06 at 9 51 57 p m

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!
- [ ] Followed the PR title naming convention mentioned above.

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

[GOV-570]: https://mifosforge.jira.com/browse/GOV-570?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by fynmanoj** [comment]
@alberto-art3ch , If you can send the PR title in this format:
[JIRA-Ticket] Description 

the CIRCLE CI will build this and push image to docker hub.

File: src/app/core/authentication/authentication.service.ts [hunk 1]
```diff
@@ -171,7 +171,7 @@ export class AuthenticationService {
 
       });
     } else {
-      console.log("Not Implemented " + environment.oauth.type);
+      console.log('Not Implemented ' + environment.oauth.type);
```
Comment (fynmanoj) [inline src/app/core/authentication/authentication.service.ts:174]: let's remove logs that are added for debug while development

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: src/environments/environment.ts [hunk 2]
```diff
@@ -4,26 +4,29 @@
...
   backend: {
-    operations: window['env']['serverApiUrlOps'] || "https://paymenthub.qa.oneacrefund.org/opsapp/api/v1",
-    vouchers: window['env']['serverApiUrlVou'] || "https://paymenthub.qa.oneacrefund.org/opsapp/api/v1",
-    account: window['env']['serverApiUrlAct'] || "https://paymenthub.qa.oneacrefund.org/opsapp/api/v1",
-    registeringInstituionId: window['env']['registeringInstitutionId'] || "default",
+    operations: window['env']['serverApiUrlOps'] || 'https://paymenthub.qa.oneacrefund.org/opsapp/api/v1',
+    signatureBatchKey: window['env']['signatureBatchKey'] || '',
+    bulkConnectorOps: window['env']['bulkConnectorOps'] || 'https://bulk-connector-demo.sandbox.fynarfin.io',
+    vouchers: window['env']['serverApiUrlVou'] || 'https://paymenthub.qa.oneacrefund.org/opsapp/api/v1',
+    voucherCallbackUrl: window['env']['callbackUrlVou'] || 'https://webhook.site/',
```
Comment (fynmanoj) [inline src/environments/environment.ts:18]: Please make the callback URLs configurable from backend

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: env.sample [hunk 3]
```diff
@@ -1,33 +1,36 @@
 PH_OPS_BACKEND_SERVER_URL=https://paymenthub.qa.oneacrefund.org/opsapp/api/v1
+export PH_OPS_BATCH_SIGNATURE=12345677899
```
[the remark this answers is not in the document]
Comment (alberto-art3ch) [reply env.sample:2]: Done! removed

File: env.sample [hunk 4]
```diff
@@ -1,33 +1,36 @@
...
+export PH_OPS_BULK_CONNECTOR_URL=https://paymenthub.qa.oneacrefund.org
```
[the remark this answers is not in the document]
Comment (alberto-art3ch) [reply env.sample:3]: Done! I've created a PR for that https://github.com/openMF/ph-ee-env-template/pull/404

**Comment by alberto-art3ch** [comment]
> @alberto-art3ch , If you can send the PR title in this format: [JIRA-Ticket] Description
> 
> the CIRCLE CI will build this and push image to docker hub.

Done!

**Pull request merged**